### ISSUE TITLE
Consent banner negative option link should be 44px minimum

### ISSUE THREAD

**Issue by HarryVerhoef** [BBC-archive/psammead#4336]
**Describe the bug**
The consent banner negative option (Find out what has changed/No, take me to settings) link should have a hit area of 44px (2.75rem) minimum.

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to any page using the psammead-consent-banner in an incognito window, 
2. Inspect the negative option link

**Expected behaviour**
The height of the inspected anchor to be 44px minimum

**Screenshots**
<img width="207" alt="Screenshot 2021-02-10 at 15 30 42" src="https://user-images.githubusercontent.com/40858929/107536221-702c3800-6bb9-11eb-81f5-90060ec384b6.png">
(https://www.bbc.com/pidgin)

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

**Comment by HarryVerhoef** [comment]
Modifying the height of the anchor results further complexity and unwanted behaviour regarding the `border-bottom` faux-underline, so this issue has been closed.
![image](https://user-images.githubusercontent.com/40858929/107544865-90acc000-6bc2-11eb-800f-358024926943.png)
